### PR TITLE
Create skeleton for online store

### PR DESCRIPTION
**Pull request by patrickelectric**
The idea is for people to check available apps without having a BlueOS running locally

image

### CODE REVIEW

**Comment by patrickelectric** [comment]
It appears that the router only works in dev mode but not once it's build, moved to draft once finding out whats going on

**Comment by patrickelectric** [comment]
ping @Williangalvani

File: website/src/styles/settings.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/**
```
Comment (Williangalvani) [inline website/src/styles/settings.scss:1]: is this effectively an empty file?

**Review by Williangalvani — APPROVED** [review]
This a temporary thing while we wait for Bazaar, right?
I'm not a big fan of having a website in this repo. but for now that should be fine.

File: website/src/styles/settings.scss (hunk 1 above)
Comment (patrickelectric) [reply website/src/styles/settings.scss:1]: Yes, that's part of the template provided by vuetify

**Comment by patrickelectric** [comment]
> This a temporary thing while we wait for Bazaar, right? I'm not a big fan of having a website in this repo. but for now that should be fine.

Yep, meanwhile, we can add small improvements just for people to check the available extensions and versions, without the need to have BlueOS running locally, and for developers to check if their extensions are already available on the store.

**Pull request merged**